app/screens/apps_form/apps_form_field.tsx

```diff
@@ -24,18 +24,20 @@ export type Props = {
     theme: Theme;
 
     value: AppFormValue;
-    onChange: (name: string, value: string | AppSelectOption) => void;
+    onChange: (name: string, value: string | AppSelectOption | AppSelectOption[]) => void;
     performLookup: (name: string, userInput: string) => Promise<AppSelectOption[]>;
 }
 
 type State = {
-    selected: DialogOption | null;
+    selected?: DialogOption | DialogOption[];
 }
 
 export default class AppsFormField extends React.PureComponent<Props, State> {
-    state = {
-        selected: null,
-    };
+    constructor(props:Props) {
+        super(props);
+
+        this.state = {};
+    }
```

Comment:
Typescript does not like that. It feels like doing it that way "shadows" the actual state. In practice, probably does not, but still, typescript will consider `this.state` to have type `{}` instead of type `State`.